### PR TITLE
Tdarr: Initial Role Addition

### PR DESCRIPTION
**Pull request by Visorask**
Add role to community.yml
Add role to appveyor.yml
Add role dir and main.yml

### CODE REVIEW

File: roles/tdarr/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      VIRTUAL_HOST: "tdarr.{{ user.domain }}"
+      VIRTUAL_PORT: 8265
+      LETSENCRYPT_HOST: "tdarr.{{ user.domain }}"
+      LETSENCRYPT_EMAIL: "{{ user.email }}"
+    volumes:
+      - "/mnt/unionfs/Media:/home/tdarr/media"
+      - "/opt/tdarr/documents:/home/documents/tdarr"
+      - "/opt/tdarr/db:/var/lib/mongodb"
+      - "/dev/shm:/dev/shm"
+      - "/etc/localtime:/etc/localtime:ro"
```
Comment (desimaniac) [inline roles/tdarr/tasks/main.yml]: This line needs to be removed. Dont need it now that we/you are using TZ variable.

File: roles/tdarr/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      PUID: "{{ uid }}"
+      PGID: "{{ gid }}"
+      VIRTUAL_HOST: "tdarr.{{ user.domain }}"
+      VIRTUAL_PORT: 8265
+      LETSENCRYPT_HOST: "tdarr.{{ user.domain }}"
+      LETSENCRYPT_EMAIL: "{{ user.email }}"
+    volumes:
+      - "/mnt/unionfs/Media:/home/tdarr/media"
+      - "/opt/tdarr/documents:/home/documents/tdarr"
+      - "/opt/tdarr/db:/var/lib/mongodb"
```
Comment (desimaniac) [inline roles/tdarr/tasks/main.yml]: Id create these folders before trying to mount them.

**Comment by desimaniac** [comment]
Think you need to rebase your fork with latest dev branch.

**Comment by Visorask** [comment]
Yeah not sure what happened there.

**Comment by Visorask** [comment]
I'm just gonna open new request once I get this sorted, still learning.

**Pull request closed without merging**